### ISSUE TITLE
Response 503 On User Reports

### ISSUE THREAD

**Issue by WingerOfOwl** [RecyThing/RecyThing-API#109]
It Must Be 201
![Screenshot (1670)](https://github.com/RecyThing/RecyThing-API/assets/102946487/55eff9fe-a5e6-4055-824d-650765216220)

**Comment by Trois25** [comment]
@WingerOfOwl yeah cause the file we can use for upload just .jpg, .png and .mp4
![image](https://github.com/RecyThing/RecyThing-API/assets/89327360/5caeaa95-0f53-4bc3-9814-b2ce60abd0b9)

**Comment by WingerOfOwl** [comment]
> @WingerOfOwl yeah cause the file we can use for upload just .jpg, .png and .mp4 ![image](https://private-user-images.githubusercontent.com/89327360/287414319-5caeaa95-0f53-4bc3-9814-b2ce60abd0b9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0ODI5NzEsIm5iZiI6MTcwMTQ4MjY3MSwicGF0aCI6Ii84OTMyNzM2MC8yODc0MTQzMTktNWNhZWFhOTUtMGY1My00YmMzLTk4MTQtYjJjZTYwYWJkMGI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAyVDAyMDQzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZjNDM4NDAyNTM1NDY1Y2JmY2Q4Zjg0MmVmMWJlODllNGM1YzJiODhmNDUxYzE4ZDllYWNkNDEwZmZkZGJhYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8FFDMgQO32jnpBoR09JraI9TaOtBPfDOg9KAuCKrdRM)

Ahh I see that, Ok over all it save if we note the type of file. Thanks juan

**Comment by WingerOfOwl** [comment]
@

**Comment by WingerOfOwl** [comment]
@Trois25 please add error handle when I Send with JPEG Format

**Comment by Trois25** [comment]
@WingerOfOwl ow thats right, after i finished my work right now ill make it

**Comment by WingerOfOwl** [comment]
> @WingerOfOwl ow thats right, after i finished my work right now ill make it

Ok, I'll wait

**Comment by Trois25** [comment]
@WingerOfOwl i already done fix it, maybe just wait for the server to update

**Comment by WingerOfOwl** [comment]
@Trois25 there is different error now :)
![Screenshot (1696)](https://github.com/RecyThing/RecyThing-API/assets/102946487/40445303-ea62-481b-90dd-69df75a80a33)

can you fix it with 400 response ?

**Comment by teranixbq** [comment]
![Screenshot from 2023-12-03 09-22-22](https://github.com/RecyThing/RecyThing-API/assets/66883583/fafa637f-bd3d-412d-a768-3bc64b43c378)

done, 500 >> 400

**Comment by WingerOfOwl** [comment]
ok, bug fixed
